config.py

```diff
@@ -41,8 +41,8 @@ class Config(object):
                                              RAS_DJANGO_SERVICE_HOST,
                                              RAS_DJANGO_SERVICE_PORT)
 
-    RAS_API_GATEWAY_SERVICE_HOST = os.getenv('RAS_API_GATEWAY_SERVICE_HOST', 'localhost')
-    RAS_API_GATEWAY_SERVICE_PORT = os.getenv('RAS_API_GATEWAY_SERVICE_PORT', 8083)
-    RAS_API_GATEWAY_SERVICE = '{}://{}:{}'.format(RAS_PROTOCOL,
-                                                  RAS_API_GATEWAY_SERVICE_HOST,
-                                                  RAS_API_GATEWAY_SERVICE_PORT)
+    RAS_FRONTSTAGE_API_SERVICE_HOST = os.getenv('RAS_FRONTSTAGE_API_SERVICE_HOST', 'localhost')
+    RAS_FRONTSTAGE_API_SERVICE_PORT = os.getenv('RAS_FRONTSTAGE_API_SERVICE_PORT', 8083)
+    RAS_FRONTSTAGE_API_SERVICE = '{}://{}:{}'.format(RAS_PROTOCOL,
```

Comment:
We should probably just have one environmental variable for the apis. Something like RAS_FRONTSTAGE_API = os.getenv('RAS_FRONTSTAGE_API', 'http://localhost:8083'). I think it's unnecessary to have each part of the url split up

Reply:
👍 This would help to cut back on deployment config overrides as well. I'll add to the TODO list.